cmd/agent.go

```diff
@@ -57,4 +57,12 @@ func init() {
 		"",
 		"Location of the credentials file. For OAuth2 based authentication.",
 	)
+	agentCmd.PersistentFlags().BoolVarP(
+		&agent.OneShot,
+		"one-shot",
+		"o",
```

Comment:
`-o` is often used for `--output`. I don't know what `--output` would do for the agent, but if we add it later
then it would be unexpected for it not to be `-o`.

I don't think that we need a short option for `--one-shot`, so I would just have the long option here, with
no short option.